internal/v1/api.yaml

```diff
@@ -1215,18 +1215,22 @@ components:
       required:
         - architecture
         - image_type
-        - upload_request
```

Comment:
Hm, why remove this as required? Image requests are useless without an upload_request.